### PR TITLE
fix build warnings

### PR DESCRIPTION
**Pull request by michaelkirk**
comments in individual commits

PTAL @charlesmchen

### CODE REVIEW

File: SignalMessaging/categories/UIView+OWS.m [hunk 1]
```diff
@@ -252,12 +253,11 @@ - (void)centerOnSuperview
 
 - (BOOL)isRTL
 {
-    if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(9, 0)) {
+    if (@available(iOS 9.0, *)) {
```
Comment (charlesmchen) [inline SignalMessaging/categories/UIView+OWS.m]: Let's make sure we do a solid iOS 8 test pass for this release.

File: SignalServiceKit.podspec [hunk 2]
```diff
@@ -27,6 +27,12 @@ An Objective-C library for communicating with the Signal messaging service.
   s.requires_arc = true
   s.source_files = 'SignalServiceKit/src/**/*.{h,m,mm}'
 
+  # We want to use modules to avoid clobbering CocoaLumberjack macros defined
+  # by other OWS modules which *also* import CocoaLumberjack. But because we
+  # also use Objective-C++, modules are disabled unless we explicitly enable
+  # them
```
Comment (charlesmchen) [inline SignalServiceKit.podspec]: Missing trailing period? Does this fix the build warnings?

**Comment by charlesmchen** [comment]
> comments in individual commits

@michaelkirk I don't see any comments on the commits.  Does GH discard them when you rebase?

**Comment by michaelkirk** [comment]
> @michaelkirk I don't see any comments on the commits. Does GH discard them when you rebase?
 Delete branch

Sorry I meant to say:

My commits were intended to be semantically complete. If the changes didn't make sense in aggregate, you can refer to the individual commit messages/diffs for a more fine grained explanation.

**Pull request merged**